app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -16,4 +19,16 @@ class HelpActivity : InjectableAppCompatActivity() {
     helpActivityPresenter.handleOnCreate()
     title = getString(R.string.menu_help)
   }
+
+  companion object {
+    fun createHelpActivityIntent(context: Context, profileId: Int?): Intent {
+      val intent = Intent(context, HelpActivity::class.java)
+      intent.putExtra(KEY_HOME_PROFILE_ID, profileId)
+      return intent
+    }
+
+    fun getIntentKey(): String {
+      return KEY_HOME_PROFILE_ID
```

Comment:
This ID is incorrect. Create `KEY_HELP_PROFILE_ID`

Reply:
Also, by the way, help does not need profile id.

Reply:
There is this line in the `NavigationDrawerFragmentPresenter`, 
`internalProfileId = activity.intent.getIntExtra(KEY_HOME_PROFILE_ID, -1)`, which uses this parameter no matter what activity, drawer is opened from, so I kept the same parameter name in `HelpActivity `. The HelpActivity needs profileId to display the Admin. Controls option if the user is admin when the nav. drawer is opened from it.

Reply:
I would suggest remove the "profileId" parameter from `createHelpActivityIntent` and then follow that error and fix in navigation drawer. We don't need profileId in HelpActivity

Reply:
@rt4914 in HelpActivity, we want to show Admin. Controls option when nav. drawer is opened, if the user is admin. For that the nav. drawer, when opened from HelpActivity, requires ProfileId to check whether the user is admin or not. That's why it is needed.